eodag/resources/providers.yml

```diff
@@ -3961,1324 +3961,888 @@
+    SEASONAL_POSTPROCESSED_PL:
+      productType: EO:ECMWF:DAT:SEASONAL_POSTPROCESSED_PRESSURE_LEVELS
+      providerProductType:
+        - ensemble_mean
+      variable:
+        - geopotential_anomaly
+        - specific_humidity_anomaly
+        - temperature_anomaly
+        - u_component_of_wind_anomaly
+        - v_component_of_wind_anomaly
+      pressure_level:
+        - "10"
+        - "30"
+        - "50"
+        - "100"
+        - "200"
+        - "300"
+        - "400"
+        - "500"
+        - "700"
+        - "850"
+        - "925"
+        - "1000"
+      leadtime_month:
+        - "1"
+      origin: ecmwf
+      system: "5"
+      format: grib
       metadata_mapping:
-        id:
-          - '{{"stringChoiceValues": [{{"name": "type", "value": "P4_1B_LR_____"}}, {{"name": "timeliness", "value": "NT"}}, {{"name": "sat", "value": "Sentinel-6A"}}], "dateRangeSelectValues": [{{"name": "position", "start": {id#split_id_into_s3_params}["startDate"], "end": {id#split_id_into_s3_params}["endDate"]}}]}}'
-          - '{$.productInfo.product#remove_extension}'
+        id: '$.id'
         providerProductType:
-          - '{{"stringChoiceValues": [{{"name": "type", "value": "{providerProductType}"}}]}}'
-          - '$.null'
-        platform:
-          - '{{"stringChoiceValues": [{{"name": "sat", "value": "{platform}"}}]}}'
-          - '$.null'
-        timeliness:
-          - '{{"stringChoiceValues": [{{"name": "timeliness", "value": "{timeliness}"}}]}}'
+          - '{{"productType": {providerProductType}}}'
           - '$.null'
-        relativeOrbitNumber:
-          - '{{"stringInputValues": [{{"name": "relorbit", "value": "{relativeOrbitNumber}"}}]}}'
-          - '$.null'
-        orbitNumber:
-          - '{{"stringInputValues": [{{"name": "orbit", "value": "{orbitNumber}"}}]}}'
-          - '$.null'
-        cycleNumber:
-          - '{{"stringInputValues": [{{"name": "cycle", "value": "{cycleNumber}"}}]}}'
+        <<: *month_year
+        origin:
+          - '{{"originating_centre": "{origin}"}}'
           - '$.null'
-        defaultGeometry: 'POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))' # geometry is not available for this product type
-    S6_P4_L1BAHR_F06:
-      productType: EO:EUM:DAT:0238
-      platform: Sentinel-6A
-      providerProductType: P4_1B_HR_____
-      timeliness: NT
+        defaultGeometry: 'POLYGON((180 -90, 180 90, -180 90, -180 -90, 180 -90))'
+        orderLink: 'https://gateway.prod.wekeo2.eu/hda-broker/api/v1/dataaccess/download?{{"location": "{downloadLink}","product_id":"{id}", "cacheable": "true", "dataset_id": "EO:ECMWF:DAT:SEASONAL_POSTPROCESSED_PRESSURE_LEVELS"}}'
+    SEASONAL_MONTHLY_SL:
+      productType: EO:ECMWF:DAT:SEASONAL_MONTHLY_SINGLE_LEVELS
+      variable:
+        - 10m_u_component_of_wind
+        - 10m_wind_gust_since_previous_post_processing
+        - 10m_v_component_of_wind
```

Comment:
please use a minimal configuration with 1 value per parameter, like configured for `cop_cds`, in order to avoid very long requests when not customized